__init__.py

```diff
@@ -1024,6 +1024,9 @@ def handler(self, context):
     bpy.app.timers.register(execute_queued_functions)
     bpy.app.timers.register(uninstall_old_version_timer)
 
+    global threading_service

```

Comment:
Why is this marked as a global variable when you are getting the value from a function on the next line?

Reply:
I changed this initialize the global only.